pkgs/build-support/fetchurl/builder.sh

```diff
@@ -2,20 +2,23 @@ source $stdenv/setup
 
 source $mirrorsFile
 
+curlVersion=$(curl -V | head -1 | cut -d' ' -f2)
```

Comment:
I'm doing this instead of `curl.version` to support native `stdenv` (e.g. on Darwin).